modules/lightning_features/lightning_core/modules/lightning_roles/config/install/lightning_roles.settings.yml

```diff
@@ -8,6 +8,9 @@ content_roles:
       - 'view ? revisions'
       - 'view own unpublished content'
       - 'create url aliases'
+      - 'access in-place editing'
+      - 'access contextual links'
+      - 'access toolbar'
```

Comment:
This is just housecleaning? I can understand Quick Edit and Contextual Links because they actively involve editing, but Toolbar?